### DISCUSSION TITLE
Cannot Add Integration After HACS Installation

### DISCUSSION THREAD

**Discussion by CautiousCapsLock** [HandyHat/ha-hildebrandglow-dcc#239] [General]
After installing through HACS, I verified that the folder /config/custom_components/hilderbrandglow_dcc/ is present, but when going to HA integrations I cannot find the integration by searching. Removed the folder and reinstalled with reboots of HA before and after. Not sure if I've missed something simple.

Thanks

**Comment by HandyHat** [comment]
You may need to clear your browser cache before the integration appears in the list

Duplicate of #34

**Reply by CautiousCapsLock** [reply]
Perfect, that resolved it. Thanks.
